tests/sudoswap/ethereum/sudoswap_ethereum_assert_samples.sql

```diff
@@ -18,10 +18,10 @@ WITH trades as (
     , case when (t.platform_fee_amount = ex.platform_fee_amount) then true else false end as correct_platform_fee
     , case when (t.pool_fee_amount = ex.pool_fee_amount) then true else false end as correct_pool_fee
     from trades t
-    outer join examples ex
+    join examples ex
```

Comment:
We actually want to do outer join here because we want the the full union of the two sets.

Reply:
got it, i think syntax needs to be 'full outer join' or it fails, i can push a commit if so